nix/default.nix

```diff
@@ -16,6 +18,7 @@ let
     gdk-pixbuf
     glib
     webkitgtk_4_1
+    ./overlay.nix
```

Comment:
:eyes:

Reply:
IM A FUCKING IMBECILE AND CANT USE A KEYBOARD :trollface: